docs/pages/getting-started/providers/sailpoint.mdx

```diff
@@ -0,0 +1,176 @@
+            email: profile.email,
+            phone: profile.phone,
+            workPhone: profile.workPhone,
+            firstname: profile.firstname,
+            lastname: profile.lastname,
+            capabilities: profile.capabilities,
+            displayName: profile.displayName,
+            name: profile.uid,
+        }
+      },
+      style: { text: "#011E69", bg: "#fff", logo: "sailpoint.svg" },
+    },
+  ],
+})
+```
+
+  </Code.Next>
+  <Code.Svelte>
+
+```ts filename="/src/auth.ts"
+import { SvelteKitAuth } from "@auth/sveltekit"
+import { env } from "$env/dynamic/prviate"
+
+export const { handle, signIn, signOut } = SvelteKitAuth({
+  providers: [
+    {
+      id: "identitySecureCloud",
+      name: "Identity Secure Cloud",
+      type: "oauth",
+      clientId: env.ISC_CLIENT_ID!,
+      clientSecret: env.ISC_CLIENT_SECRET!,
+      authorization: {
+        url: `${env.ISC_BASE_URL!}/oauth/authorize`,
+        params: { scope: 'sp:scopes:all' },
+      },
+      token: `${env.ISC_BASE_API_URL!}/oauth/token`,
+      userinfo: `${env.ISC_BASE_API_URL!}/oauth/userinfo`,
+      profile(profile: IdentitySecureCloudProfile) {
+        return {
+            tenant: profile.tenant,
+            id: profile.id,
+            uid: profile.uid,
+            email: profile.email,
+            phone: profile.phone,
+            workPhone: profile.workPhone,
+            firstname: profile.firstname,
+            lastname: profile.lastname,
+            capabilities: profile.capabilities,
+            displayName: profile.displayName,
+            name: profile.uid,
+        }
+      },
+      style: { text: "#011E69", bg: "#fff", logo: "sailpoint.svg" },
+    },
+  ],
+})
+```
+
+  </Code.Svelte>
+  <Code.Express>
+
+```ts filename="/src/app.ts"
+import { ExpressAuth } from "@auth/express"
+
+app.use("/auth/*", ExpressAuth({ providers: [
+  {
+      id: "identitySecureCloud",
+      name: "Identity Secure Cloud",
+      type: "oauth",
+      clientId: process.env.ISC_CLIENT_ID!,
+      clientSecret: process.env.ISC_CLIENT_SECRET!,
+      authorization: {
+        url: `${process.env.ISC_BASE_URL!}/oauth/authorize`,
+        params: { scope: 'sp:scopes:all' },
+      },
+      token: `${process.env.ISC_BASE_API_URL!}/oauth/token`,
+      userinfo: `${process.env.ISC_BASE_API_URL!}/oauth/userinfo`,
+      profile(profile: IdentitySecureCloudProfile) {
+        return {
+            tenant: profile.tenant,
+            id: profile.id,
+            uid: profile.uid,
+            email: profile.email,
+            phone: profile.phone,
+            workPhone: profile.workPhone,
+            firstname: profile.firstname,
+            lastname: profile.lastname,
+            capabilities: profile.capabilities,
+            displayName: profile.displayName,
+            name: profile.uid,
+        }
+      },
+      style: { text: "#011E69", bg: "#fff", logo: "sailpoint.svg" },
+    },
+] }))
+```
+
+  </Code.Express>
+</Code>
```

Comment:
And somethign like this below the example code to point folks in the right direction, should they want to use any of the additional fields

```suggestion
</Code>

Your `userprofile` endpoint will return more fields, but by default the [User table](https://authjs.dev/getting-started/database#models) only supports `id`, `name`, `email`, and `image`. Therefore, if you'd like to use any of the following fields, make sure you modify the `User` table schema in whichever adapter / database you're using.

	```ts
	tenant: profile.tenant,
	id: profile.id,
	uid: profile.uid,
	email: profile.email,
	phone: profile.phone,
	workPhone: profile.workPhone,
	firstname: profile.firstname,
	lastname: profile.lastname,
	capabilities: profile.capabilities,
	displayName: profile.displayName,
	name: profile.uid
	```

The above fields will all be available in the `profile` callback.


```

Reply:
Thanks a lot, this looks perfect, in my case, I also did the same which extend the User Type and also User table schema